.drone.star

```diff
@@ -34,6 +34,7 @@ REDIS = "redis:6-alpine"
 SELENIUM_STANDALONE_CHROME = "selenium/standalone-chrome:104.0-20220812"
 SONARSOURCE_SONAR_SCANNER_CLI = "sonarsource/sonar-scanner-cli:4.7.0"
 THEGEEKLAB_DRONE_GITHUB_COMMENT = "thegeeklab/drone-github-comment:1"
+OC_CI_CLAMAVD = "owncloudci/clamavd"
```

Comment:
Not a big issue but in the past we arrange variables in alphabetical order maybe it will be nice if we follow that

Reply:
yes that's for reminding me